src/components/common/Directors.vue

```diff
@@ -601,7 +601,7 @@ export default class Directors extends Mixins(DateMixin, ObjectMixin, DirectorMi
       addressCountry: '',
       deliveryInstructions: ''
     },
-    appointmentDate: this.asOfDate,
```

Comment:
This is another "property is used before being initialized" TS error, fixed with a watcher.